ios/FluentUI/People Picker/PeoplePicker.swift

```diff
@@ -214,7 +215,7 @@ open class PeoplePicker: BadgeField {
 
     // MARK: Personas
 
-    private func showPersonaSuggestions() {
+    open func showPersonaSuggestions() {
```

Comment:
>open [](start = 4, length = 4)

please add documentation to all public/open vars/conts/methods.

Reply:
added